package.json

```diff
@@ -16,9 +15,10 @@
     "prettier": "prettier './**/*.{js,jsx,ts,tsx,json,yml,yaml,css,md}' --write",
     "lint": "eslint './packages/**/*.{js,jsx,ts,tsx}'",
     "clean": "npm run clean-logs && npm run clean-modules",
-    "clean-logs": "rimraf ./packages/*/npm-debug* && rimraf ./*-debug*",
+    "clean-logs": "rimraf ./packages/*/*-debug* && rimraf ./*-debug*",
     "clean-modules": "rimraf ./packages/*/node_modules/ && rimraf ./node_modules/",
-    "release": "yarn radix:build && lerna publish"
+    "release": "lerna publish --conventional-commits",
+    "release:github": "lerna publish --github-release --conventional-commits"
```

Comment:
Think we should remove for now 👍 As per my other comments above

Reply:
Yeah if you want man!

I don't want to wrangle code management from you for this repo, and would rather your decisions weigh to the final choice.

Keep in mind though that penned GitHub Releases are probably easier to reach than the individual `CHANGELOG` files in the packages. In the effort to be clear to folks that considering using the component library, or are already using it, having the most accessible changelog is attractive.